docker/Dockerfile.cpu

```diff
@@ -0,0 +1,19 @@
+FROM python:3.11
+
+RUN apt-get update && apt-get install -y \
+	ffmpeg
+
+WORKDIR /app
```

Comment:
WORKDIR /roop

Reply:
It is needed for the following ADD commands to copy the files to the correct directory